### PR TITLE
Add ktlint-wrapper pre-commit hook to template

### PR DESCRIPTION
**Pull request by emmaworley**
**Description of changes**
Add ktlint-wrapper pre-commit hook to template and `pre-commit run --all-files` to test script

**Testing done**
`./test.sh` passes locally

### CODE REVIEW

**Comment by emmaworley** [comment]
CI failed because dev.imjac.in is down, @andycate can you re-run it?

File: {{cookiecutter.repo_name}}/src/test/kotlin/tests/Testy.kt [hunk 1]
```diff
@@ -1,10 +1,7 @@
 package tests
 
-import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
 
-import frc.team{{cookiecutter.team_number}}.{{cookiecutter.repo_name}}.Robot
```
Comment (andycate) [inline {{cookiecutter.repo_name}}/src/test/kotlin/tests/Testy.kt:6]: weren't we gonna have some initial test so that it is obvious how to use tests?

**Review by andycate — COMMENTED** [review]
In Testy.kt, I thought we were gonna have an initial test that used Robot.kt.

File: {{cookiecutter.repo_name}}/src/test/kotlin/tests/Testy.kt (hunk 1 above)
Comment (emmaworley) [reply {{cookiecutter.repo_name}}/src/test/kotlin/tests/Testy.kt:6]: Yeah, ktlint complains about this being an unused import though. We should add it back in another PR with those example tests.

Comment (emmaworley) [reply {{cookiecutter.repo_name}}/src/test/kotlin/tests/Testy.kt:6]: FYI the ktlint output will look something like this when things fail:
```
ktlint wrapper...........................................................Failed
hookid: ktlint-wrapper

src/main/kotlin/frc/team5499/example_robot/Robot.kt:1:1: Package name must not contain underscore (cannot be auto-corrected) (package-name)
src/test/kotlin/tests/Testy.kt:3:1: Unused import (no-unused-imports)
src/test/kotlin/tests/Testy.kt:6:1: Unused import (no-unused-imports)
```

Comment (andycate) [reply {{cookiecutter.repo_name}}/src/test/kotlin/tests/Testy.kt:6]: Ok, for sure

**Pull request merged**